### ISSUE TITLE
Pin moves away from correct coordinates in Add Location view when zooming out

### ISSUE THREAD

**Issue by shafy** [mapzy/mapzy#37]
See how this is implemented in [`map_controller.js`](https://github.com/mapzy/mapzy/blob/main/app/javascript/controllers/map_controller.js) - we need to do the same in 
`location_controller.js` to prevent that the pin floats away when zooming out.

**Comment by sagzy** [comment]
Which part exactly?

```
      new mapboxgl.Marker({ 
        element: anchor,
        // offset is necessary because we are using a custom element
        offset: [-12, -35]
      })
        .setLngLat(feature.geometry.coordinates)
        .addTo(this.map);
    }
```

Is it the `offset`?

**Comment by shafy** [comment]
Good question. Not sure if it's the offset. I thought that was only necessary because we construct the element in this hacky way to work with Turbo Frames. But this is not what we do in `location_controller.js`.

Maybe we need to define a different anchor instead of `"bottom"`?

```js
this.marker = new mapboxgl.Marker({
  draggable: true,
  color: "#E74D67",
  anchor: "bottom"
});
```

Need to play around and see.

**Comment by sagzy** [comment]
Great intuition mister. Fixed it by removing the `anchor` attribute #42
